lib/slim.js

```diff
@@ -1,59 +1,30 @@
 const isWsl = require('is-wsl');
+const glob = require('glob-all');
+const fse = require('fs-extra');
 
-/**
- * Get commands to slim the installed requirements
- * only for non-windows platforms:
- * works for docker builds and when run on UNIX platforms (wsl included)
- * @param {Object} options
- * @param {string} folderPath
- * @return {Array.<String>}
- */
-function getSlimPackageCommands(options, folderPath) {
-  let stripCmd = [];
+const getStripCommand = (options, folderPath) =>
+  process.platform !== 'win32' || isWsl || options.dockerizePip
+    ? ` && find ${folderPath} -name "*.so" -exec strip {} \\;`
+    : '';
 
-  // Default stripping is done for non-windows environments
-  if (process.platform !== 'win32' || isWsl) {
-    stripCmd = getDefaultSLimOptions(folderPath);
-
-    // If specified any custom patterns to remove
-    if (options.slimPatterns instanceof Array) {
-      // Add the custom specified patterns to remove to the default commands
-      const customPatterns = options.slimPatterns.map(pattern => {
-        return getRemovalCommand(folderPath, pattern);
-      });
-      stripCmd = stripCmd.concat(customPatterns);
+const deleteFiles = (options, folderPath) => {
+  let patterns = [
+    '**/*.so',
```

Comment:
Also deleteing the [`.so` files](https://docs.python.org/3/extending/building.html) might not necessarily be safe as these are [extension modules](https://wiki.python.org/moin/PythonPackagingTerminology).

Reply:
DoH! Good catch. I didn't mean to delete those. that's what `strip` is for!